presidio-analyzer/presidio_analyzer/nlp_engine/transformers_nlp_engine.py

```diff
@@ -57,8 +57,9 @@ def __call__(self, doc: Doc) -> Doc:
         ents = []
         for d in res:
             span = doc.char_span(d["start"], d["end"], label=d["entity_group"])
-            span._.confidence_score = d["score"]
-            ents.append(span)
+            if span is not None:
```

Comment:
Regarding to Spacy doc: https://spacy.io/api/doc#char_span, `char_span` can return None with "strict" mode

Reply:
Referencing to this #911. If you're already at it, could you please change the `alignment_mode` to `expand`

Reply:
This is fixed in #941